assets/schema/ensemble_schema.json

```diff
@@ -2819,9 +2819,17 @@
                       "type": "boolean",
                       "description": "Determines if carousel should loop infinitely or be limited to item length. Default (false)"
                     },
-                    "selectedItemIndex": {
+                    "selectedIndex": {
```

Comment:
I thought we agreed one will be internal and the other to be public. I'm looking at this as a customer and I don't understand the difference between selectedIndex and currentIndex.

Reply:
`currentIndex` - Which will be used for the first and the second point above. 0 is the default value if it's not given by the user
`selectedIndex` - It'll return the value only if the card selected or else it'll have -1 as the value